### DISCUSSION TITLE
Problems with Math-Rendering

### DISCUSSION THREAD

**Discussion by EinMaulwurf** [danny-avila/LibreChat#1772] [Troubleshooting]
### What happened?

There are problems with rendering the following section:
"""
1. **Normalize the distribution**: Convert the $N(\mu, \sigma^2)$ distribution to the standard normal distribution $N(0,1)$. The formula for converting an arbitrary value $x$ from a normal distribution $N(\mu, \sigma^2)$ to a standard normal distribution $z$ is:

   $$
   z = \frac{x - \mu}{\sigma}
   $$

   where $\mu$ is the mean and $\sigma$ is the standard deviation of the original distribution. Note that the standard deviation $\sigma$ is the square root of the variance, so in your case, $\sigma = \sqrt{25} = 5$.

2. **Find the quantile in the standard normal distribution**: Since you need the $0.5$ quantile (which corresponds to the median) of the original distribution and the standard normal distribution is symmetric about $0$, the $0.5$ quantile of the standard normal distribution is $0$ (because the mean of the standard normal distribution is $0$).

3. **Convert back to the original distribution**: Use the formula to convert the $z$ value back to the original scale using 

   $$
   x = z\sigma + \mu
   $$

   Since $z = 0$ for the $0.5$ quantile in the standard normal distribution, substituting $\mu=3$ and $\sigma=5$, we get:

   $$
   x_{0.5} = 0 \cdot 5 + 3 = 3
   $$

"""
It turns out like this:
<img width="657" alt="Bildschirmfoto 2024-02-11 um 12 42 06" src="https://github.com/danny-avila/LibreChat/assets/116873121/5f4d1d20-0e4f-473e-82d9-a3a8092a03c4">

Some inline and display math is rendered correctly, but there are problems with other inline-expressions.

### Steps to Reproduce

Ask the assistant to repeat word-by-word the following section:
```
1. **Normalize the distribution**: Convert the $N(\mu, \sigma^2)$ distribution to the standard normal distribution $N(0,1)$. The formula for converting an arbitrary value $x$ from a normal distribution $N(\mu, \sigma^2)$ to a standard normal distribution $z$ is:

   $$
   z = \frac{x - \mu}{\sigma}
   $$

   where $\mu$ is the mean and $\sigma$ is the standard deviation of the original distribution. Note that the standard deviation $\sigma$ is the square root of the variance, so in your case, $\sigma = \sqrt{25} = 5$.

2. **Find the quantile in the standard normal distribution**: Since you need the $0.5$ quantile (which corresponds to the median) of the original distribution and the standard normal distribution is symmetric about $0$, the $0.5$ quantile of the standard normal distribution is $0$ (because the mean of the standard normal distribution is $0$).

3. **Convert back to the original distribution**: Use the formula to convert the $z$ value back to the original scale using 

   $$
   x = z\sigma + \mu
   $$

   Since $z = 0$ for the $0.5$ quantile in the standard normal distribution, substituting $\mu=3$ and $\sigma=5$, we get:

   $$
   x_{0.5} = 0 \cdot 5 + 3 = 3
   $$
```

### What browsers are you seeing the problem on?

Firefox

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Native LaTeX formatting is supported now, so try wraping with \[ or \( instead of the markdown flavor `$$` or `$`
